README.md

```diff
@@ -58,6 +58,7 @@ The following are all _required_.
 |`projects`|Space-separated list of paths of projects. When omitted, falls back to the environment variable `SENTRY_PROJECT` to determine the project.|-|
 |`url_prefix`|Adds a prefix to source map urls after stripping them.|-|
 |`strip_common_prefix`|Will remove a common prefix from uploaded filenames. Useful for removing a path that is build-machine-specific.|`false`|
```

Comment:
Can you replace these two lines with this:
```
|`strip_prefix`|Strips the given prefix from uploaded filenames. Useful for removing a path that is build-machine-specific.|-|
|`strip_common_prefix`|Similar to `strip-prefix` but strips the most common prefix on all sources references when set to `true`.|`false`|
```
This order matches the `sentry-cli` documentation.